main.go

```diff
@@ -80,9 +81,6 @@ func run(ctx context.Context) error {
 		return errors.Wrap(err, "failed to initialise service")
 	}
 
-	// The following value could be read from some a config setting ...
-	debug.SetGCPercent(25)
-
```

Comment:
Why is this being removed?

Reply:
This has never made it to prod, and it was a guess at an attempt to rectify the performance issues.